css/style.css

```diff
@@ -112,8 +113,19 @@
   vertical-align: middle;
   opacity: .5;
 }
+.svg-star {
+  opacity: 0.3;
+}
+.svg-star:hover {
+  opacity: 0.5;
+}
+.svg-star-low,
+.svg-star-medium,
+.svg-star-high {
+  opacity: 1 !important;
+}
 .icon.none {
-  background-position: 0px -20px;
```

Comment:
if this is used for icons where no background should be shown, I propose `background-image: unset;` as a more permanent solution